lib/cache/Cache.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path')
 const mkdirp = require('mkdirp')
 const os = require('os')
+const fs = require('fs')
```

Comment:
I think we use ‘fs-extra’ elsewhere, maybe we should use that for consistency?  Might not be an issue since I think ‘fs’ is used under the hood for that lib

Reply:
It's not in the `package.json` for `cli-helpers-engine`; though `fs-extra` replaces both `mkdirp` and `rimraf`, so I can refactor them to use it and swap deps.